### DISCUSSION TITLE
How to join the Tables In apache superset i have 2 excel tables it have common fields are there i want join is there any proper document to do that

### DISCUSSION THREAD

**Discussion by Vijay4171** [apache/superset#19958] [Q&A / Help]
### Expected results

what you expected to happen.

### Actual results

what actually happens.

### Environment

(please complete the following information):

- browser type and version:
- superset version: `superset version`
- python version: `python --version`
- node.js version: `node -v`
- any feature flags active:

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by zhaoyongjie** [comment]
Currently, Superset can't load users' data, so you can not directly join 2 Excel files. The workaround is that upload the Excel file into the database, then use SQLLab to make a join table.
